tests/BaseTestCase.php

```diff
@@ -28,30 +28,4 @@ protected function defineEnvironment($app)
             'prefix'   => '',
         ]);
     }
-
-    /**
-     * Create a fake cache entry for a given request.
-     */
-    protected static function mockRequestCachedResponse(
```

Comment:
These should be in a trait that consumers (like Onshop) can import them, instead of re-implementing them.